Library/Formula/influxdb.rb

```diff
@@ -81,7 +90,12 @@ def install
     Language::Go.stage_deps resources, buildpath/"src"
 
     cd influxdb_path do
-      system "go", "install", "-ldflags", "-X main.version 0.9.0 -X main.commit 471117ce5b308e59f8f247d8a88a52ead553b602", "./..."
+      if build.stable?
+        system "go", "install", "-ldflags", "-X main.version 0.9.0 -X main.commit 471117ce5b308e59f8f247d8a88a52ead553b602", "./..."
+      end
```

Comment:
Let's use an `else` statement here:

```ruby
if build.head?
  something
else
  something_else
end
```

If/When `devel` gets re-added at some point the `else` statement becomes `elsif` and it keeps the blocks a little neater.